commands/svc.cmd

```diff
@@ -18,6 +18,28 @@ DOCKER_COMPOSE_ARGS=()
 DOCKER_COMPOSE_ARGS+=("-f")
 DOCKER_COMPOSE_ARGS+=("${WARDEN_DIR}/docker/docker-compose.yml")
 
+
+## check if dnsmasq is enabled
+if [[ -f "${WARDEN_HOME_DIR}/.env" ]]; then
+    eval "$(grep "^WARDEN_DNSMASQ_ENABLE" "${WARDEN_HOME_DIR}/.env")"
+fi
+
+## add dnsmasq docker-compose
+WARDEN_DNSMASQ_ENABLE="${WARDEN_DNSMASQ_ENABLE:-1}"
+if [[ "$WARDEN_DNSMASQ_ENABLE" == "1" ]]; then
+    echo "adding docker-compose.dnsmasq.yml"
```

Comment:
I think this is useless message
```suggestion
```

Reply:
Echo statement removed. Sorry for taking so long, this somehow slipped my mind.